src/main/java/salesforce/entities/constants/ContactConstant.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.entities.constants;
+
+/**
+ * Manages contact constant information.
+ *
+ * @author Juan Martinez.
+ * @version 1.0 20 March 2020.
+ */
+public final class ContactConstant {
+    public static final String FIRST_NAME = "FirstName";
+    public static final String LAST_NAME = "LastName";
+    public static final String SALUTATION = "Salutation";
+    public static final String TITLE = "Title";
+    public static final String DEPARTMENT = "Department";
+    public static final String BIRTH_DATE = "Birthdate";
+    public static final String LEAD_SOURCE = "LeadSource";
+    public static final String PHONE = "Phone";
+    public static final String HOME_PHONE = "HomePhone";
+    public static final String MOBILE_PHONE = "MobilePhone";
+    public static final String EMAIL = "Email";
+    public static final String ASSISTANT_NAME = "AssistantName";
+    public static final String DESCRIPTION = "Description";
+
+    /**
+     * ContactConstant constructor.
+     */
+    private ContactConstant() {
+
+    }
```

Comment:
It is necessary?, java have a default empty constructor

Reply:
yes because it is a final class and it should not be instantiate

Reply:
Its private, sorry I coudn't se XD

Reply:
ok dont worry